libs/watt/src/lib/components/field/watt-field.component.scss

```
@@ -76,11 +76,21 @@ watt-field {
 
   .label {
     min-height: 27px;
-    display: block;
+    display: flex;
+    align-items: center;
     padding-bottom: var(--watt-space-s);
 
+    watt-icon {
+      margin-left: var(--watt-space-s);
+    }
+
+    watt-tooltip {
+      width: 250px;
```

suggestion: in order to account for less amount of text in the popup, let's use `max-width` instead.

```suggestion
      max-width: 250px;
```